fosspay/setup.py

```diff
@@ -0,0 +1,42 @@
+"""
+setup.py
+Stripe Payments Demo. Created by Adrienne Dreyfus (@adrind).
+
+This is a one-time setup script for your server. It creates a set of fixtures,
+namely products and SKUs, that can then used to caluclate payment amounts when completing the
+checkout flow in the web interface.
+"""
+
+import stripe
+import os
+from dotenv import load_dotenv, find_dotenv
+
+load_dotenv(find_dotenv())
+
+stripe.api_key = _cfg("stripe-secret")
+stripe.api_version = '2019-03-14'
+
+
+def create_data():
```

Comment:
is this data used anywhere? The inventory doesn't seem to be related to any of the projects listed on the payment page.